modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/side_panel/SidePanel.js

```diff
@@ -382,21 +385,29 @@ export default class SidePanel extends React.Component {
 						/>
 					)}
 
-					<ClayButton
-						aria-label={Liferay.Language.get('close')}
-						className={classNames(
-							'fds-side-panel-close',
-							this.state.closeButtonStyle === 'simple' &&
-								'fds-side-panel-close-simple',
-							this.state.closeButtonStyle === 'menu' &&
-								'fds-side-panel-close-menu'
-						)}
-						displayType="monospaced"
-						onClick={() => this.close()}
-						title={Liferay.Language.get('close')}
-					>
-						<ClayIcon symbol="times" />
-					</ClayButton>
+					{this.state.title && (
+						<div className="fds-side-panel-header">
+							<div className="fds-side-panel-title">
+								<h3 className="mb-0">{this.state.title}</h3>
+							</div>
+
+							<ClayButton
+								aria-label={Liferay.Language.get('close')}
+								className={classNames(
+									'fds-side-panel-close',
+									this.state.closeButtonStyle === 'simple' &&
+										'fds-side-panel-close-simple',
+									this.state.closeButtonStyle === 'menu' &&
+										'fds-side-panel-close-menu'
+								)}
+								displayType="monospaced"
+								onClick={() => this.close()}
+								title={Liferay.Language.get('close')}
+							>
+								<ClayIcon symbol="times" />
+							</ClayButton>
+						</div>
+					)}
```

Comment:
All of this is [added by ClayHeader](https://github.com/liferay/clay/blob/master/packages/clay-modal/src/Header.tsx#L127-L143), in a standard way. Do we need the special styles? If yes, can we add them in a CSS file, and override default implementation?

Reply:
Custom side panel implementations with title + close button (Commerce and Objects) have different styles than the default one, so I tried to unify all of them. I could remove them and rely on the default styles.